### ISSUE TITLE
API Error

### ISSUE THREAD

**Issue by kennygokh** [qgrail/obsidian-ai-assistant#5]
Hi, when I try to run AI assistance, it promoted API error. But there is no error code so not sure what is the issue. Please help.

**Comment by qgrail** [comment]
Hello @kennygokh,
I have updated the plugin to display the API error explicitly. 
Can you update to the latest version and tell me what your error is?

**Comment by CalAlaera** [comment]
I started a chat with the AI Assistant just now and gave it the prompt "What should my daily goal be for today?". I received the following error:

`## OpenAI API ## SyntaxError: Expected property name or '}' in JSON at position 1`

I have entered the correct API key in the settings menu.

**Comment by lalas** [comment]
Same here.

**Comment by eriksr** [comment]
Same issue as CalAlaera, here. The plugin is not usable at all as a result.

**Comment by GiacomoLaw** [comment]
Same issue here.

**Comment by KreigD** [comment]
Also having this issue.

**Comment by oramadn** [comment]
Same here!

**Comment by paulpiazza** [comment]
same here!

**Comment by AnonMall** [comment]
same for me

**Comment by PhoenixBodrov** [comment]
> Same issue as CalAlaera, here. The plugin is not usable at all as a result.

Yes, i also have API errors, sent screenshots in a separate issue (maybe i should've posted them here... )

**Comment by ngacho** [comment]
Shameless plug: For those comfortable with (or willing to try) running a local llm server, I forked this project [here](https://github.com/ngacho/obsidian-ai-assistant) and I'm currently using it instead of this plugin. Of course thanks to [qgrail](https://github.com/qgrail) for building this, but I couldn't make it work so I opted for a local llm server. I'm open to contributions for adding stable diffusion and whisper support without relying on any online apis.

**Comment by jpreslov** [comment]
also having this issue. installed from the Community Plugins page

**Comment by sam-mic** [comment]
Same error here, is it because the OpenAI server break down? Today is a big day for ChatGPT+. I tried to ask in  official website but failed

**Comment by krelltunez** [comment]
Anyone find a fix for this?  I hadn't used this in a while, but tried tonight and I'm getting this same error.

**Comment by PhoenixBodrov** [comment]
NO! That's the whole problem. Also if you find it - t.me/fenixrein

ср, 29 нояб. 2023 г. в 05:20, krelltunez ***@***.***>:

**Comment by MickToews** [comment]
Have this issue: ## OpenAl API ## Error: 404 The model
*gpt-4' does not exist or you do not have access to it. Learn more:
https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4."

But I have gpt4 Version
